packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -40,6 +45,8 @@ export abstract class TableColumn<
      */
     public readonly columnInternals: ColumnInternals<TColumnConfig>;
 
+    protected validator: Validator;
```

Comment:
We're working pretty hard to keep internal properties off the TableColumn prototype chain / keep internal properties configured through column internals objects so column authors know what to configure (notice we removed other abstract properties, methods, overridable properties, etc.)

I think a better alternative would be to include a ColumnValidator as an optional option in ColumnInternalsOptions and if it's not provided create a default instance. 

Then the column author can manipulate `columnInternals.validator` in a way that is typed to their implementation.

Reply:
We discussed this and came to the decision that the validator should be a private field on the table column.